spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -585,4 +585,30 @@
       expect(presenter.as_json[:attributes][:recruited_at]).to be_present
     end
   end
+
+  describe 'compliance with models that change updated_at' do
+    it 'looks at all fields which cause a touch' do
+      choice = create(:application_choice, :awaiting_provider_decision)
+
+      # if there is a field on the form that causes a touch but isn't
+      # queried via the presenter on the API, fail this test
+      (ApplicationForm::PUBLISHED_FIELDS - %w[international_address right_to_work_or_study_details]).each do |field|
+        expect_any_instance_of(ApplicationForm).to receive(field).at_least(:once).and_call_original
```

Comment:
RuboCop doesn't like us doing this. I'm not sure what the cleanest way to get around this would be @duncanjbrown ?

Reply:
It is dirty! We can disable RuboCop for just these lines: https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code